trikot-viewmodels-declarative-flow/compose-tv-flow/src/main/kotlin/com/mirego/trikot/viewmodels/declarative/compose/viewmodel/VMDButtonTv.kt

```diff
@@ -0,0 +1,61 @@
+package com.mirego.trikot.viewmodels.declarative.compose.viewmodel
+
+import androidx.compose.foundation.interaction.MutableInteractionSource
+import androidx.compose.foundation.interaction.collectIsFocusedAsState
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.RowScope
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import androidx.tv.material3.Button
+import androidx.tv.material3.ButtonBorder
+import androidx.tv.material3.ButtonColors
+import androidx.tv.material3.ButtonDefaults
+import androidx.tv.material3.ButtonGlow
+import androidx.tv.material3.ButtonScale
+import androidx.tv.material3.ButtonShape
+import androidx.tv.material3.ExperimentalTvMaterial3Api
+import com.mirego.trikot.viewmodels.declarative.components.VMDButtonViewModel
+import com.mirego.trikot.viewmodels.declarative.compose.extensions.isOverridingAlpha
+import com.mirego.trikot.viewmodels.declarative.compose.extensions.observeAsState
+import com.mirego.trikot.viewmodels.declarative.compose.extensions.vmdModifier
+import com.mirego.trikot.viewmodels.declarative.content.VMDContent
+
+@ExperimentalTvMaterial3Api
+@Composable
+fun <C : VMDContent> VMDButtonTv(
```

Comment:
Should we put these views in their own package instead of suffixing them with Tv? We could reference them using only the import (like the material 3 version of the views vs the standard ones)